app/companies/my-profile/page.tsx

```diff
@@ -0,0 +1,260 @@
+"use client";
+
+import { useRef, useState, FormEvent } from "react";
+import DragAndDropFile from "../../components/drag-and-drop-file";
+import { Button } from "../../components/button";
+import toast from "react-hot-toast";
```

Comment:
Put node modules import at first, just after `React`.

Add `react-hot-toast` in packages.